pkg/query/logical/plan_indexscan_local.go

```diff
@@ -173,6 +180,10 @@ func (i *localIndexScan) executeInShard(shard tsdb.Shard) ([]tsdb.Iterator, erro
 		return nil, err
 	}
 
+	if seriesList == nil || len(seriesList) == 0 {
```

Comment:
`len()` can handle both empty and nil slices,

Reply:
Thx! But it is strange. Why didn't the linter report this issue?

Reply:
Fixed